collector/src/main/java/org/apache/hertzbeat/collector/util/CollectUtil.java

```diff
@@ -309,7 +311,7 @@ public static JsonElement replaceSmilingPlaceholder(JsonElement jsonElement, Map
                         Map<String, String> map = JsonUtil.fromJson(jsonValue, typeReference);
                         if (map != null) {
                             map.forEach((name, value) -> {
-                                if (name != null && !"".equals(name.trim())) {
+                                if (StringUtils.isEmpty(name)) {
```

Comment:
hi, here is not empty name

Reply:
Thanks! @tomsun28 
I have fixed it!!! 👍